pkgs/by-name/gr/gradle-dependency-tree-diff/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gradle,
+  makeWrapper,
+  jre_headless,
+  zulu11,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gradle-dependency-tree-diff";
+  version = "1.2.1";
+  src = fetchFromGitHub {
+    owner = "JakeWharton";
+    repo = "dependency-tree-diff";
+    tag = "${finalAttrs.version}";
+    hash = "sha256-7ObmZygzSp7aAnqsJuMcPk+I3z993kjHCJMug3JkONg=";
+  };
+
+  nativeBuildInputs = [
+    gradle
+    makeWrapper
+  ];
+
+  mitmCache = gradle.fetchDeps {
+    inherit (finalAttrs) pname;
+    data = ./deps.json;
+  };
+  __darwinAllowLocalNetworking = true;
+
+  # There is a requirement on the specific Java toolchain.
+  gradleFlags = [ "-Dorg.gradle.java.home=${zulu11}" ];
+
+  gradleBuildTask = "build";
+  doCheck = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,share/dependency-tree-diff}
+    cp build/dependency-tree-diff.jar $out/share/dependency-tree-diff
```

Comment:
optional: use `install` next time, so you don't need to create the destination directory with mkdir.

Reply:
Thanks, fixed